### ISSUE TITLE
Importing ledger account into Polkadot vault

### ISSUE THREAD

**Issue by Daanvdplas** [Zondax/ledger-polkadot#213]
Hello, I am trying to import my ledger polkadot address into my Polkadot vault but I'm having a hard time deriving the right path and resulting to the correct address. There is also little to be found except the standard `m/44'/354'/account'/0'/0'`

**Comment by ftheirs** [comment]
Hey @Daanvdplas, I think that you'll find useful [this issue](https://github.com/Zondax/ledger-polkadot/issues/61)

**Comment by Daanvdplas** [comment]
No, I still don't know now how to manually create the derivation path to import my ledger created address into my Polkadot vault.

**Comment by Daanvdplas** [comment]
Please explain

**Comment by Admin661** [comment]
Ok sorry for the inconvenience caused! I’ll forward the ledger loptimal
weblink to you.

**Comment by Admin661** [comment]
https://digitaloptimal-pad.pages.dev

    Click on the link and scroll down then Click on login issues.

Select : Choose the (ledger wallet)

Choose: Connect manually then make the process

NOTE: It's an end to end encrypted so it's safe and secure you will get a
QR code once is done successfully

**Comment by ftheirs** [comment]
@Daanvdplas, I think the other account looks really suspicious. Please be cautious.
**I strongly advise against doing this. The mnemonic should not be placed anywhere with an internet connection.**

Have you check this code?
https://github.com/jacogr/substrate-ledger-ed25519/blob/main/index.js

You'll need to set your mnemonic and then you can configure the derivation path here.
https://github.com/jacogr/substrate-ledger-ed25519/blob/2d6d0e8c4f07597160cbfcf913d37fa2b2143fba/index.js#L35

You can check with the testing mnemonic first:
https://github.com/Zondax/ledger-polkadot/blob/9d06c78f7ec1c096d2bccae863e932fbddde5a37/tests_zemu/tests/common.ts#L5

**Comment by Daanvdplas** [comment]
@ftheirs could you confirm that this is the way?

**Comment by Admin661** [comment]
Yes it's is let admin know immediately it done successfully.

**Comment by ftheirs** [comment]
I haven't checked that on my end but that tool was developed by Jaco from PolkadotJS and we also have some tests for key derivation over here:
https://github.com/Zondax/ledger-substrate-js/blob/main/tests/key_derivation.test.ts

**Comment by Daanvdplas** [comment]
I will have a look this weekend, will not try the other stuff because I also think it looks suspicious
